ThermofluidStream/package.mo

```diff
@@ -4,22 +4,48 @@ package ThermofluidStream "Library for the Modelling of Thermofluid Streams"
 
   import Modelica.Units.SI;
 
-annotation (
-  version="1.0.0",
-  uses(
-    Modelica(version="4.0.0"),
-    Complex(version="4.0.0")),
-  conversion(
-    noneFromVersion="0.4.1",
-    from(
-      version="0.2.1",
-      to="0.3.0",
-      script="modelica://ThermofluidStream/Resources/ConversionSkripts/convert0_2_1to0_3_0.mos")),
-  Documentation(info="<html>
+  annotation (
+    version = "1.0.0",
+    versionDate = "2022-12-02",
+    dateModified = "2022-12-02 11:00:00Z",
+    revisionId = "$Format:%h %ci$",
```

Comment:
Can you explain how the ID (commit hash?) of the revision will be inserted here?

Reply:
Okay then it seems not to work properly...

Yes I agree with the versionDate. So does it mean I have to set the dateModified manually for every new build?

Reply:
Yes - as far as I know.